Castle.Core.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30011.22
+# Visual Studio Version 17
+VisualStudioVersion = 17.11.35312.102
```

Comment:
I've since converted the solution file to the simpler `.slnx` format, please edit `Castle.Core.slnx` instead.

Reply:
Have merged into `Castle.Core.slnx` instead.